### ISSUE TITLE
Dashboard Story Template

### ISSUE THREAD

**Issue by jagus7** [CBIIT/nci-ctd2-dashboard-tmpl-helper#55]
Please add the following URL to Dashboard story template on the Submission Builder:

https://ocg.cancer.gov/ctd2-dashboard-stories-template

**Comment by zhouji2013** [comment]
Should this be added as another button besides 'Manual', "Manuscript', and 'Glossary'? If so, what is the proper label for the button? Or somewhere else?

**Comment by kcs3** [comment]
Yes let's just put it in a new button called "Stories" between "Manual" and "Manuscript".  Let's add hover text to it that reads "Preparing a story submission".

While we are at it, we can add hover text to the other buttons too:
Manual - "Submission Builder Manual"
Manuscript - "CTD^2 Dashboard publication"
Glossary - "CTD^2 Dashboard term definitions.
